packages/legacy-workbench/src/js/angular/core/directives/yasgui-component/yasgui-component.directive.js

```diff
@@ -423,6 +422,10 @@ function yasguiComponentDirective(
                         YasguiPersistenceMigrationService.migrateYasguiPersistence($translate.instant('sparql.tab.directive.unnamed.tab.title'));
                     }
 
+                    if (!config.themeName) {
+                        config.themeName = ThemeService.isDarkModeApplied() ? 'moxer': null;
```

Comment:
Can you encapsulate this moxer theme name in the ThemeService, e.g. create some constant there and a getter for the dark theme name?

Reply:
Done